christfellowshipchurch/web-app-v2
# updated web branding

### About
This ticket updates web branding

Changes that were made across the website:
1. Updated logo
2. Updated CF icon
3. Updated colors (brand colors, system colors, and neutrals)

### Test Instructions
See the screenshots before and feel free to open the Netlify link to test all of the changes made across the website

### Screenshots

1. Home page / footer
<img width="1439" alt="Screen Shot 2021-07-23 at 4 57 10 PM" src="https://user-images.githubusercontent.com/46769629/126842418-3c4189ef-db5e-4722-969f-3acf1e7b83dd.png">
<img width="1421" alt="Screen Shot 2021-07-23 at 4 57 31 PM" src="https://user-images.githubusercontent.com/46769629/126842426-039172e2-dd77-43ff-8465-ae7a3d8e51b3.png">

2. A Message/sermon/article 
<img width="1313" alt="Screen Shot 2021-07-23 at 4 58 53 PM" src="https://user-images.githubusercontent.com/46769629/126842454-3adf58fd-0989-47bf-a363-0be469e25c91.png">

3. An Event
<img width="1440" alt="Screen Shot 2021-07-23 at 4 58 19 PM" src="https://user-images.githubusercontent.com/46769629/126842463-474a7899-edc9-4be1-ae1c-001fadf839e6.png">

4. Profile Screen
<img width="1430" alt="Screen Shot 2021-07-23 at 5 00 00 PM" src="https://user-images.githubusercontent.com/46769629/126842666-f19f417a-1fa6-4cb5-8b23-b65a3c1c176b.png">
<img width="1173" alt="Screen Shot 2021-07-23 at 4 59 52 PM" src="https://user-images.githubusercontent.com/46769629/126842671-d847c5d4-7cd4-4021-add7-a5917d61eded.png">

5. A single Group
<img width="1322" alt="Screen Shot 2021-07-23 at 5 00 18 PM" src="https://user-images.githubusercontent.com/46769629/126842709-212d5ce6-d74e-4d2a-9fcc-791a3b82f167.png">

6. Groups Search
<img width="1429" alt="Screen Shot 2021-07-23 at 5 12 35 PM" src="https://user-images.githubusercontent.com/46769629/126842726-552707db-f8c5-4ade-b25c-620e5b0c464f.png">

7. A Group Hub page
<img width="1439" alt="Screen Shot 2021-07-23 at 5 13 54 PM" src="https://user-images.githubusercontent.com/46769629/126842776-7c49ba8a-f5a0-4f3c-9ed8-598445e8dd13.png">
<img width="1341" alt="Screen Shot 2021-07-23 at 5 00 43 PM" src="https://user-images.githubusercontent.com/46769629/126842782-19cc7717-3349-4c12-a179-60a26df8507a.png">

### Closes Tickets
[CFDP-1590]

### Related Tickets
[CFDP-1589]


[CFDP-1590]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1590
